shared/src/native-src/pal.h

```diff
@@ -155,20 +169,22 @@ inline WSTRING GetCurrentProcessCommandLine()
             cmdline.append(buf, len);
         }
     }
+    fclose(fp);
```

Comment:
I think the close should be in the `if (fp)` block?

Not that it matter much since it was already outside previously.

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/5564/commits/8023556a751e21444a1311d923ca971ab9dbfd98